komga-webui/src/locales/en.json

```diff
@@ -203,12 +202,12 @@
     "nothing_to_show": "Nothing to show",
     "outdated": "Outdated",
     "pages": "pages",
-    "pages_n": "No pages | 1 page | {count} pages",
+    "pages_n": "No pages | One page | {count} pages",
```

Comment:
the figure notation is important for display size, "One" in other languages could be very long, and overflow.

Reply:
`max-length:10`? https://docs.weblate.org/en/latest/user/checks.html?highlight=flag#maximum-length-of-translation

Reply:
it wouldn't be consistent with the plural form, so you would have `one page`, `2 pages`. I prefer `1 page`.

Reply:
So it is less often wrong.
Also, it is "No pages", not "0 pages"